doc/002-local-dev.md

```diff
@@ -1,184 +1,207 @@
 
-### Multi-Platform Linux Builder Container
+## Build Applications Individually
 
-The purpose of this container is to document the process of building locally the Linux executables in the same way as the GitHub Actions workflow (CI) which automation is not accessible to downstream contributors. By default, this produces three executables for each Ziti component, one for each platform architecture: amd64, arm, arm64. You may instead build for one or more of these by specifying the architecture as a parameter to the `docker run` command as shown below.
+### `ziti` CLI
 
-#### Build the Container Image
+```bash
+# build the binary without installing in GOPATH
+go build -o ./build/ziti ./ziti/cmd/ziti/
+# execute the binary
+./build/ziti version
+```
 
-You only need to build the container image once unless you change the Dockerfile or `./linux-build.sh` (the container's entrypoint).
+### `ziti-controller`
 
 ```bash
-# find the latest Go distribution's semver
-LATEST_GOLANG=$(curl -sSfL "https://go.dev/VERSION?m=text" | /bin/grep -Po '^go(\s+)?\K\d+\.\d+\.\d+$')
-# build a container image named "zitibuilder" with the Dockerfile in the top-level of this repo
-docker build \
-    --tag=zitibuilder \
-    --file=Dockerfile.linux-build \
-    --build-arg latest_golang=${LATEST_GOLANG} \
-    --build-arg uid=$UID \
-    --build-arg gid=$GID .
+# build the binary without installing in GOPATH
+go build -o ./build/ziti-controller ./ziti-controller/
+# execute the binary
+./build/ziti-controller version
 ```
 
-#### Run the Container to Build Executables for the Desired Architectures
-
-Executing the following `docker run` command will:
-1. Mount the top-level of this repo on the container's `/mnt`
-2. Run `./linux-build.sh ${@}` inside the container
-3. Deposit built executables in `./release`
+### `ziti-router`
 
 ```bash
-# build for all three architectures: amd64 arm arm64
-docker run \
-    --rm \
-    --name=zitibuilder \
-    --volume=$PWD:/mnt \
-    zitibuilder
-
-# build only amd64 
-docker run \
-    --rm \
-    --name=zitibuilder \
-    --volume=$PWD:/mnt \
-    zitibuilder \
-        amd64
+# build the binary without installing in GOPATH
+go build -o ./build/ziti-router ./ziti-router/
+# execute the binary
+./build/ziti-router version
 ```
 
-You will find the built artifacts in `./release`.
+## Crossbuilds
+
+When you push to your repo fork then GitHub Actions will automatically crossbuild for several OSs and CPU architectures. You'll then be able to download the built artifacts from the GitHub UI. The easiest way to crossbuild the Linux exectuables locally is to build and run the crossbuild container. Please refer to [the crossbuild container README](../Dockerfile.linux-build.README) for those steps. For hints on crossbuilding for MacOS and Windows see [the main GitHub Actions workflow](../.github/workflows/main.yml) which defines the steps that are run when you push to GitHub.
```

Comment:
this should all be 'anywhere else'. This isn't relevant to "local development". This all feels way more related to publishing than local dev

Reply:
Moved to BUILD.md